[2025-07-06T14:54:46.069+0000] {logging_mixin.py:154} INFO - Changing /opt/***/logs/dag_id=turnstile_ml_pipeline/run_id=scheduled__2025-07-05T00:00:00+00:00/task_id=download_and_ingest permission to 509
[2025-07-06T14:54:46.070+0000] {logging_mixin.py:154} INFO - Failed to change /opt/***/logs/dag_id=turnstile_ml_pipeline/run_id=scheduled__2025-07-05T00:00:00+00:00/task_id=download_and_ingest permission to 509: [Errno 1] Operation not permitted: '/opt/***/logs/dag_id=turnstile_ml_pipeline/run_id=scheduled__2025-07-05T00:00:00+00:00/task_id=download_and_ingest'
[2025-07-06T14:54:46.151+0000] {logging_mixin.py:154} INFO - Changing /opt/***/logs/dag_id=turnstile_ml_pipeline/run_id=scheduled__2025-07-05T00:00:00+00:00/task_id=download_and_ingest permission to 509
[2025-07-06T14:54:46.152+0000] {logging_mixin.py:154} INFO - Failed to change /opt/***/logs/dag_id=turnstile_ml_pipeline/run_id=scheduled__2025-07-05T00:00:00+00:00/task_id=download_and_ingest permission to 509: [Errno 1] Operation not permitted: '/opt/***/logs/dag_id=turnstile_ml_pipeline/run_id=scheduled__2025-07-05T00:00:00+00:00/task_id=download_and_ingest'
[2025-07-06T14:54:46.243+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: turnstile_ml_pipeline.download_and_ingest scheduled__2025-07-05T00:00:00+00:00 [queued]>
[2025-07-06T14:54:46.258+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: turnstile_ml_pipeline.download_and_ingest scheduled__2025-07-05T00:00:00+00:00 [queued]>
[2025-07-06T14:54:46.259+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 1
[2025-07-06T14:54:46.279+0000] {taskinstance.py:1382} INFO - Executing <Task(BashOperator): download_and_ingest> on 2025-07-05 00:00:00+00:00
[2025-07-06T14:54:46.289+0000] {standard_task_runner.py:57} INFO - Started process 804 to run task
[2025-07-06T14:54:46.321+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'turnstile_ml_pipeline', 'download_and_ingest', 'scheduled__2025-07-05T00:00:00+00:00', '--job-id', '3', '--raw', '--subdir', 'DAGS_FOLDER/pipeline_dag.py', '--cfg-path', '/tmp/tmpnjohk4fg']
[2025-07-06T14:54:46.334+0000] {standard_task_runner.py:85} INFO - Job 3: Subtask download_and_ingest
[2025-07-06T14:54:46.424+0000] {logging_mixin.py:154} WARNING - /home/***/.local/lib/python3.8/site-packages/***/settings.py:195 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2025-07-06T14:54:46.620+0000] {logging_mixin.py:154} INFO - Changing /opt/***/logs/dag_id=turnstile_ml_pipeline/run_id=scheduled__2025-07-05T00:00:00+00:00/task_id=download_and_ingest permission to 509
[2025-07-06T14:54:46.622+0000] {logging_mixin.py:154} INFO - Failed to change /opt/***/logs/dag_id=turnstile_ml_pipeline/run_id=scheduled__2025-07-05T00:00:00+00:00/task_id=download_and_ingest permission to 509: [Errno 1] Operation not permitted: '/opt/***/logs/dag_id=turnstile_ml_pipeline/run_id=scheduled__2025-07-05T00:00:00+00:00/task_id=download_and_ingest'
[2025-07-06T14:54:46.626+0000] {task_command.py:416} INFO - Running <TaskInstance: turnstile_ml_pipeline.download_and_ingest scheduled__2025-07-05T00:00:00+00:00 [running]> on host 77d8d9060233
[2025-07-06T14:54:47.312+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='turnstile_ml_pipeline' AIRFLOW_CTX_TASK_ID='download_and_ingest' AIRFLOW_CTX_EXECUTION_DATE='2025-07-05T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-07-05T00:00:00+00:00'
[2025-07-06T14:54:47.315+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2025-07-06T14:54:47.319+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', '\n            python /opt/***/pipeline/download_ingest_turnstile.py                 --year { ds_year }                 --month { ds_month }                 --user None                 --password None                 --host None                 --port None                 --db_name None                 --table_name raw_turnstile_data\n        ']
[2025-07-06T14:54:47.338+0000] {subprocess.py:86} INFO - Output:
[2025-07-06T14:54:47.724+0000] {subprocess.py:93} INFO - python: can't open file '/opt/***/pipeline/download_ingest_turnstile.py': [Errno 2] No such file or directory
[2025-07-06T14:54:47.730+0000] {subprocess.py:97} INFO - Command exited with return code 2
[2025-07-06T14:54:47.761+0000] {taskinstance.py:1937} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/bash.py", line 210, in execute
    raise AirflowException(
airflow.exceptions.AirflowException: Bash command failed. The command returned a non-zero exit code 2.
[2025-07-06T14:54:47.771+0000] {taskinstance.py:1400} INFO - Marking task as FAILED. dag_id=turnstile_ml_pipeline, task_id=download_and_ingest, execution_date=20250705T000000, start_date=20250706T145446, end_date=20250706T145447
[2025-07-06T14:54:47.814+0000] {standard_task_runner.py:104} ERROR - Failed to execute job 3 for task download_and_ingest (Bash command failed. The command returned a non-zero exit code 2.; 804)
[2025-07-06T14:54:47.857+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 1
[2025-07-06T14:54:47.972+0000] {taskinstance.py:2778} INFO - 0 downstream tasks scheduled from follow-on schedule check
